Emby.Server.Implementations/HttpServer/HttpListenerHost.cs

```diff
@@ -566,9 +539,7 @@ protected async Task RequestHandler(IHttpRequest httpReq, string urlString, stri
                     return;
                 }
 
-                if (string.Equals(localPath, "/mediabrowser/", StringComparison.OrdinalIgnoreCase) ||
-                    string.Equals(localPath, "/mediabrowser", StringComparison.OrdinalIgnoreCase) ||
-                    localPath.IndexOf("mediabrowser/web", StringComparison.OrdinalIgnoreCase) != -1)
+                if (localPath.IndexOf("mediabrowser/web", StringComparison.OrdinalIgnoreCase) != -1)
```

Comment:
why drop first two?

Reply:
They will never evaluate to true as they are already checked before this.
https://github.com/jellyfin/jellyfin/pull/455/files/99d47963b438005c938ddbaaa052589132ec24fd#diff-234173e74ae9494988401966dcd4f002R530

Reply:
my bad... thanks!